react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+// function Button (props) {
+//   return (
+//     <button ClassName={classColor} >Click Me!</button>
+//   );
+// }
+
+
+class HotButton extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = { click: 0}
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick(){
+    this.setState({ click: this.state.click + 1 })
+    console.log("work")
+  }
+
+  render (){
+
+    // const Button = <button className={color}>Hot Button</button>;
+    if(this.state.click == 0 ||  this.state.click <= 2){
+      return (
```

Comment:
Well done on this Brian! The next step is to move the logic so deciding the classname to another method which will return the correct classname. Then you can call it in the render method, assign the proper classname to a variable, and them use that variable to assign the classname to the button. This will really clean up the render method, and prevent you from having to return many versions of almost the same JSX